server/container/guice/mailbox-plugin-deleted-messages-vault-guice/src/main/java/org/apache/james/modules/vault/BlobStoreDeletedMessageVaultModule.java

```diff
@@ -31,10 +32,13 @@
 
     @Override
     protected void configure() {
-        bind(MemoryDeletedMessageMetadataVault.class).in(Scopes.SINGLETON);
+        install(new CassandraDeletedMessageMetadataVaultModule());
```

Comment:
The general Module should not deal with Cassandra one, it's the other way around.